contents/blog/how-to-measure-product-market-fit.md

```diff
@@ -0,0 +1,332 @@
+---
+date: 2023-06-08
+title: "In-depth: How to reliably measure product-market fit"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["andy-vandervell"]
+featuredImage: ../images/blog/pmf-game-guide/pmf-guide.png
+featuredImageType: full
+category: Startups
+tags:
+  - Guides
+---
+
+[Achieving product-market fit](/blog/product-market-fit-game) isn't a golden ticket to a billion-dollar IPO, but startups can't succeed without it. 
+
+Most guides tell you how it _feels_ to have product-market fit. Everything feels effortless when you have it, or impossibly difficult when you don't, and so on.
+
+But product-market fit isn't just an ephemeral gut feeling. You can _measure_ it, and _it moves_ as your customer's needs change, or new solutions appear.
+
+This guide is about finding ways to measure product-market fit, your progress toward it, and how it changes over time.
+
+## Leading and lagging indicators
```

Comment:
I really enjoyed reading the sections on each of individual metrics, but I felt that this section dragged on a bit. It reads a bit bland, like a generic blog that's trying to explain to me what product metrics are. Can we shorten it so we get to the good stuff sooner?

Perhaps cutting out the parts where it mentions the differences from Leading and Lagging indicators – it's not clear to me why I would need to know the difference?